### PR TITLE
Fix wrong gem require

### PR DESCRIPTION
**Pull request by purbon**
Fix mocha as a dependency require on the specs. , otherwise they are not being executed and fail.

### CODE REVIEW

File: logstash-output-graphite.gemspec [hunk 1]
```diff
@@ -23,5 +23,6 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'logstash', '>= 1.4.0', '< 2.0.0'
 
   s.add_development_dependency 'logstash-devutils'
+  s.add_runtime_dependency 'mocha'
```
Comment (ph) [inline logstash-output-graphite.gemspec]: This `mock` library should be declared as a development dependencies.

**Comment by jsvd** [comment]
there are tons of problems w/ the tests in this plugin, raised in #1 
- missing logstash plugins kv and generator for development
- running `bundle exec rspec -t socket --order default` halts at some point probably trying to send events to a closed port or something
- I think some tests are order dependent, need more info..

this pr might be a place to fix this stuff

**Comment by purbon** [comment]
@jsvd This is awesome, but we need to fix that in a new PR. For now we just need to fix the dependency and be able to run all test together.

can we merge this and fix the test in another PR?

**Comment by purbon** [comment]
@jsvd refactored and fixed the test after also @electrical requested. now they run properly, next step is to be sure they are testing what mathers :+1:

**Comment by electrical** [comment]
LGTM

File: spec/outputs/graphite_spec.rb [hunk 2]
```diff
@@ -1,14 +1,10 @@
-require "logstash/devutils/rspec/spec_helper"
-require "logstash/outputs/graphite"
-require "mocha/api"
+require 'spec_helper'
 
 describe LogStash::Outputs::Graphite, :socket => true do
```
Comment (jsvd) [inline spec/outputs/graphite_spec.rb]: with this patch we can remove the `:socket => true` !!

**Comment by jsvd** [comment]
LGTM

**Comment by ph** [comment]
Yes :heart_eyes:

File: spec/server.rb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    def stop
+      @server.close
+    end
+
+    def empty?
+      @queue.empty?
+    end
+
+  end
+end
```
Comment (ph) [inline spec/server.rb:39]: I think this file should be in a support folder?

**Pull request closed without merging**